### DISCUSSION TITLE
alttab

### DISCUSSION THREAD

**Discussion by aloispichler** [hyprwm/Hyprland#3169] [General]
Does Hyprland have alttab (window switching) functionality, which includes the history of opened windows?
Examples:
* https://github.com/sagb/alttab (on X11)
* https://alt-tab-macos.netlify.app/

**Comment by vaxerski** [comment]
No, that would be for you to make yourself. I believe there was some project that had that (sfwbar?) but you can also make it with elkowar's eww for example
